### PR TITLE
Adding script to download/install oras

### PR DESCRIPTION
**Pull request by juliusl**
Also, Setting the release version to "0.11.0-experimental"

### CODE REVIEW

**Comment by juliusl** [comment]
@shizhMSFT  @sajayantony

File: internal/version/version.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package version
 
 var (
 	// Version is the current version of the oras.
-	Version = "0.11.1"
+	Version = "0.11.0-experimental"
```
Comment (sajayantony) [inline internal/version/version.go:5]: @SteveLasker should we rename this to alpha similar to distribution?

File: scripts/get-oras.sh [hunk 2]
```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# This script find the release by tag, downloads it to the user's home directory
```
Comment (northtyphoon) [inline scripts/get-oras.sh:3]: finds

File: internal/version/version.go (hunk 1 above)
Comment (shizhMSFT) [reply internal/version/version.go:5]: Should it be `0.11.1-experimental`?

File: scripts/get-oras.sh [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    log "${CYAN}$*${NC}"
+}
+
+# Setup logging for troubleshooting
+logsetup
+
+# Search Parameters
+REPO='oras-project/oras'
+TAG='v0.11.0' # TODO: This tag will need to change later
+OS='linux'    # TODO: Can add OS/Architecture detection later
```
Comment (shizhMSFT) [inline scripts/get-oras.sh:40]: How to override those environment variables?

Comment (juliusl) [reply scripts/get-oras.sh]: So the way to do this now is: 
`REPO='juliusl/oras' TAG='v0.11.5-alpha' ./get-oras.sh;. ./install-oras.sh`

or from the repo 

`curl https://raw.githubusercontent.com/juliusl/oras/juliusl/prototype-3/scripts/get-oras.sh | REPO='juliusl/oras' TAG='v0.11.5-alpha'  sh;. ./install-oras.sh`

**Comment by juliusl** [comment]
@sajayantony  Could you give this a shot: 

curl https://raw.githubusercontent.com/juliusl/oras/juliusl/prototype-3/scripts/get-oras.sh | REPO='juliusl/oras' TAG='v0.11.5-alpha' sh;. ./install-oras.sh

File: cmd/oras/pull.go [hunk 4]
```diff
@@ -19,6 +19,7 @@ type pullOptions struct {
 	allowedMediaTypes  []string
 	allowAllMediaTypes bool
 	allowEmptyName     bool
+	handleEmptyName    bool
```
Comment (sajayantony) [inline cmd/oras/pull.go:22]: I'm guessing these came in from another PR.

**Comment by juliusl** [comment]
@sajayantony @northtyphoon do we not want this ?

**Comment by sajayantony** [comment]
Given the move to just curling the binary and feedback from Lachie about
not piping shell maybe we should avoid it. But I know helm does have this
in the instructions so not sure why we would not take this in.

@stevelasker should we do this?

**Comment by juliusl** [comment]
@sajayantony  Ok well if you guys want it, someone can copy and paste it from here: 

https://github.com/juliusl/azorasrc/tree/main/scripts

I have documentation. One thing I want to mention as well, is that this is actually a mini package manager as well, so you can have SxS oras binaries. 

Also, re: piping shell. For this script I do not require elevated permissions, nor do I touch anything in root. It's non destructive and adds files. That being said it comes from the internet so YMMV. (rustup also pipes shell btw)

**Pull request closed without merging**